09-secrets.md

```diff
@@ -56,4 +62,7 @@ not managed by compose lifecycle, Compose implementations SHOULD reject the Comp
 
```

Comment:
I'm not sure what is meant on line 60 - would be good to make this clearer. @milas could you clarify for me?

Reply:
means having `external: true` makes attributes other than `name` irrelevant and should trigger an error

Reply:
Thanks Nicolas, have made this clearer